README.rst

```diff
@@ -125,6 +125,22 @@ The default keys and values are specified below. If you want to use custom confi
     api = ADDRESS_OF_THE_SERVER # example: http://fancyUrlName.com:8080
     
 
+System failures
+~~~~~~~~~~~~~~~
+System failures are part of any software system. Upon some certain outputs (e.g. receiving error 500 from remote pipeline),
+we return :code:`None` in the output of call. When processing big documents it might make sense to check take care of
```

Comment:
This scenario is fine. But when the server returns error 500, we should return None and move on after showing a warning msg (instead of throwing an exception); because this is not user's issue to take care of the server-side issues.